build/charts/yorkie-analytics/README.md

```
@@ -0,0 +1,68 @@
+```bash
+helm upgrade [RELEASE_NAME] yorkie-team/yorkie-analytics -n analytics
+```
+
+With Helm v3, CRDs created by this chart are not updated by default and should be manually updated.
+Consult also the [Helm Documentation on CRDs](https://helm.sh/docs/chart_best_practices/custom_resource_definitions).
+
+_See [`helm upgrade`](https://helm.sh/docs/helm/helm_upgrade/) for command documentation._
+
+## Configuration
+
+See [Customizing the Chart Before Installing](#customizing-the-chart-before-installing) below for configuration options. To see all configurable options with detailed comments:
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid
null

(MD051, link-fragments)